src/locale/messages.en-US.xlf

```diff
@@ -402,6 +402,42 @@
         <source>notification-icon.toggle-panel.tooltip</source>
         <target state="new">Click to toggle notification panel</target>
       </trans-unit>
+      <trans-unit id="3299373246271290710" datatype="html">
+        <source>action-triggers.action.label</source>
+        <target state="new">Action</target>
+      </trans-unit>
+      <trans-unit id="719177318571875364" datatype="html">
+        <source>common.form-required-field.button-tooltip</source>
+        <target state="new">You have not filled in a required field</target>
+      </trans-unit>
+      <trans-unit id="5742386463554387927" datatype="html">
+        <source>action-triggers.shortcut.error</source>
+        <target state="new">Shortcut is required field</target>
```

Comment:
I'd change this to:
`A shortcut is required`

Otherwise this should at least be:
`Shortcut is a required field`

Reply:
And should end with a period, since it is a sentence :D

Reply:
Changed